book/src/validator-manager-move.md

```diff
@@ -0,0 +1,181 @@
+# Moving Validators
+
+The `lighthouse validator-manager move` command uses the VC HTTP API to move
+validators from one VC (the "src" VC) to another VC (the "dest" VC). The move
+operation is *comprehensive*; it will:
+
+- Disable the validators on the src VC.
+- Remove the validator keystores from the src VC file system.
+- Export the slashing database records for the appropriate validators from the src VC to the dest VC.
+- Enable the validators on the dest VC.
+- Generally result in very little or no validator downtime.
+
+It is capable of moving all validators on the src VC, a count of validators or
+a list of pubkeys.
+
+The `move` command is only guaranteed to work between two Lighthouse VCs (i.e.,
+there is no guarantee that the commands will work between Lighthouse and Teku, for instance).
+
+The `move` command only supports moving validators using a keystore on the local
+file system, it does not support `Web3Signer` validators.
+
+Although all efforts are taken to avoid it, it's possible for the `move` command
+to fail in a way that removes the validator from the src VC without adding it to the
+dest VC. Therefore, it is recommended to **never use the `move` command without
+having a backup of all validator keystores (e.g. the mnemonic).**
+
+## Simple Example
+
+The following command will move all validators from the VC running at
+`http://localhost:6062` to the VC running at `http://localhost:5062`.
+
+```bash
+lighthouse \
+	validator-manager \
+	move \
+	--src-vc-url http://localhost:6062 \
+	--src-vc-token ~/src-token.txt \
+	--dest-vc-url http://localhost:5062 \
+	--dest-vc-token ~/.lighthouse/mainnet/validators/api-token.txt \
+	--validators all
+```
+
+## Detailed Guide
+
+This guide describes the steps to move validators between two validator clients (VCs) which are
+able to SSH between each other. This guide assumes experience with the Linux command line and SSH
+connections.
+
+There will be two VCs in this example:
+
+- The *source* VC which contains the validators/keystores to be moved.
+- The *destination* VC which is to take the validators/keystores from the source.
+
+There will be two hosts in this example:
+
+- Host 1 (*"source host"*): Is running the `src-vc`.
+- Host 2 (*"destination host"*): Is running the `dest-vc`.
+
+The example will assume that all commands are run on Host 1. It also assumes
```

Comment:
This can be confusing since Steps 2 and 5 are run on Host 2 (destination host), hence the suggestion to change, and also added info in Step 4.

```suggestion
The example assumes
```